### PR TITLE
Feat(Developer guide): CI/CD introduction

### PR DESCRIPTION
**Pull request by Skraye**

### CODE REVIEW

File: docs/developer-guide/cicd/README.md [hunk 1]
```diff
@@ -4,4 +4,19 @@ order: 13
 
 # Create a CI/CD pipeline for Kestra
 
+## Introduction
+
+In Kestra, the number of flows you developed can grow quickly, and maintaining them can be a challenge.
```
Comment (tanay1337) [inline docs/developer-guide/cicd/README.md:9]:
```suggestion
In Kestra, the number of flows you develop can grow quickly, and maintaining them can get challenging.	
```

File: docs/developer-guide/cicd/README.md [hunk 2]
```diff
@@ -4,4 +4,19 @@ order: 13
...
+CI/CD pipelines are a great way to automate the validation and the deployment of your flows.
```
Comment (tanay1337) [inline docs/developer-guide/cicd/README.md:10]:
```suggestion
CI/CD pipelines are a great way to automate your flows' validation and deployment.
```

File: docs/developer-guide/cicd/README.md [hunk 3]
```diff
@@ -4,4 +4,19 @@ order: 13
...
+This section will guide you through the steps to create a CI/CD pipeline.
```
Comment (tanay1337) [inline docs/developer-guide/cicd/README.md:11]:
```suggestion
This section will guide you through creating a CI/CD pipeline for your Kestra flows.
```

File: docs/developer-guide/cicd/README.md [hunk 4]
```diff
@@ -4,4 +4,19 @@ order: 13
...
+
+## Why a CI/CD pipeline?
+
+CI/CD pipelines provide an easy and reliable way to ensure that your flows are always deployed as soon they're done.
```
Comment (tanay1337) [inline docs/developer-guide/cicd/README.md:15]: As soon as they are done with? 👀

File: docs/developer-guide/cicd/README.md [hunk 5]
```diff
@@ -4,4 +4,19 @@ order: 13
...
+It integrates well in VCS (Version Control System) like Git, and can be easily integrated in your development workflow.
```
Comment (tanay1337) [inline docs/developer-guide/cicd/README.md:16]:
```suggestion
It integrates nicely with VCS (Version Control System) like Git and can easily integrate into your development workflow.
```

**Review by tanay1337 — CHANGES_REQUESTED** [review]

File: docs/developer-guide/cicd/README.md (hunk 4 above)
Comment (loicmathieu) [reply docs/developer-guide/cicd/README.md:15]: > are always deployed as soon they're in your source control.

That's the point of a continuous integration right ? You commit, it trigger a workflow

**Review by loicmathieu — APPROVED** [review]
OK for me if you apply Tanay's suggestions and rephrase the sentence I add a comment to

File: docs/developer-guide/cicd/README.md (hunk 4 above)
Comment (tanay1337) [reply docs/developer-guide/cicd/README.md:15]: `they are in your source control.`

Yeah, that makes sense. I wasn't sure if there was more to it. Thanks :)

**Pull request merged**